src/main/resources/templates/start.html

```diff
@@ -62,63 +41,63 @@ <h2> Meal Mentor </h2>
         <tbody>
         <tr class="table-success" style="LINE-HEIGHT:60px">
             <th scope="row">Breakfast</th>
-            <td> <a href="#"> <select id="sel"> <option value=""> -- Select -- </option> </select> </a> </td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td></td>
+            <td id="breakfastM"></td>
+            <td id="breakfastT"></td>
+            <td id="breakfastW"></td>
+            <td id="breakfastR"></td>
+            <td id="breakfastF"></td>
+            <td id="breakfastS"></td>
+            <td id="breakfastU"></td>
```

Comment:
For all of the cells I added an id that does two things. 1, allows you to reference it later when you try to programmatically add text to them. 2, clearly tells you what needs to go in the cell without the need to look at the page itself.

Reply:
Good.